### PR TITLE
Workflow Automated Test - LRQA-67512 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**

### CODE REVIEW

**Comment by CalumR23** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase [hunk 1]
```diff
@@ -57,12 +57,55 @@ definition {
...
+			workflowResourceValue = "Blogs Entry");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase:80]: Can you please replace this with a Blogs API macro? 

JSONBlog.addEntry(
				entryContent = "Blogs Entry Content",
				entryTitle = "Blogs Entry Title");

**Comment by pedro-michel** [comment]
@rodrigocunhaa, can you please look at this PR? It's supposed this commit will be merged?

**Comment by rodrigocunhaa** [comment]
@pedro-michel @AfonsoHSantos , it seems to be a glitch of GitHub. The commit was not actually merged. You can ignore it.

**Pull request merged**